app/controllers/admin/reports_controller.rb

```diff
@@ -69,19 +87,25 @@ def reports
           report_title: "Application Details report ",
           report_link: { href: "https://dsdmoj.atlassian.net/wiki/x/JwBOKQE", text: "About this report (opens in new tab)", class: "govuk-link govuk-link--no-visited-state", rel: "noreferrer noopener", target: "_blank" },
           path: :admin_application_details_csv_path,
-          path_text: "Download CSV",
+          path_text: "Download <span class=\"govuk-visually-hidden\">application details</span> CSV".html_safe,
```

Comment:
One too many spaces?!
```suggestion
          path_text: "Download <span class=\"govuk-visually-hidden\">application details</span>CSV".html_safe,
```

Reply:
🤔  the space seems correct

Reply:
Because the span is hidden the output will be 
Download<space><hidden tag><space>CSV
So there will be a double white space on screen for ```Download  CSV```